src/scripts.js

```diff
@@ -74,8 +77,14 @@ function displaySelectedRecipe(e) {
     instructionsSection.innerHTML += instructionsElement
     image.src = selectedRecipe.image;
     name.innerText = selectedRecipe.name;
-  };
-};
+  }
+}
+
+function searchAllRecipes() {
```

Comment:
Is it not the greatest feeling when you write a function that invokes a method you previously wrote...and it works?!